cmake/VASTRegisterPlugin.cmake

```diff
@@ -1,5 +1,48 @@
 include_guard(GLOBAL)
 
+# Normalize the GNUInstallDirs to be relative paths, if possible.
+macro (VASTNormalizeInstallDirs)
+  foreach (
+    _install IN
+    ITEMS "BIN"
+          "SBIN"
+          "LIBEXEC"
+          "SYSCONF"
+          "SHAREDSTATE"
+          "LOCALSTATE"
+          "RUNSTATE"
+          "LIB"
+          "INCLUDE"
+          "OLDINCLUDE"
+          "DATAROOT"
+          "DATA"
+          "INFO"
+          "LOCALE"
+          "MAN"
+          "DOC")
+    # Try removing CMAKE_INSTALL_PREFIX with a trailing slash from the full path
+    # to get the correct relative path because some package managers do this
+    # stupid thing where they put absolute paths into variables that are
+    # supposed to be interpreted as relative to the install preifx.
```

Comment:
This comment is too strong. The [documentation](https://cmake.org/cmake/help/v3.21/module/GNUInstallDirs.html) explicitly states:
> ... However, an absolute path is also allowed.

In some cases absolute paths make sense, and we should probably only do this in relocatable mode for plugins too.

Reply:
I toned the comment down, and made the requested change.